packages/engine/src/scene/components/PoiUIComponent.tsx

```diff
@@ -70,6 +72,12 @@ export const PoiUIComponent = defineComponent({
 })
 
 function PoiUIReactor() {
+  const poiCameraQuery = useQuery([RendererComponent, PoiCameraComponent])
+  if (poiCameraQuery.length < 1) {
+    return null
```

Comment:
must not return before other hooks, react must have the same number of hooks across renders